Lib/compositor/font.py

```diff
@@ -6,6 +6,17 @@
 from compositor.error import CompositorError
 
 
+def _nameToString(nameRecord):
+    encoding = nameRecord.getEncoding()
+    try:
+        # python 3
+        ret = nameRecord.string.decode(encoding)
```

Comment:
actually no. If in python2 you use `tostr` on a unicode string, it will reencode it to bytes using a default ascii encoding, which is not good here.

Maybe this would be better:

```python
if str == bytes:
    # python 2
    ret = nameRecord.toBytes()
else:
    # python 3
    ret = nameRecord.toUnicode()
```